modules/virtualization/microvm/appvm.nix

```diff
@@ -27,7 +27,17 @@
           config,
           pkgs,
           ...
-        }: {
+        }: let
```

Comment:
Thanks for reviewing it. You are right, it's always best to have a single logical change per commit. In this particular case I'm not really adding support for the border color. In fact it has been there for quite a while. The problem is that the value was hardcoded in the launchers and I moved it to the appvm settings to avoid hardcoding it in the PDF script as well. That's why it's a part of this PR.